client/reader/post-byline/index.jsx

```diff
@@ -1,40 +1,63 @@
 /**
  * External Dependencies
  */
-var React = require( 'react' );
+import React from 'react';
 
 /**
  * Internal Dependencies
  */
-var ExternalLink = require( 'components/external-link' ),
-	Gravatar = require( 'components/gravatar' ),
-	Gridicon = require( 'components/gridicon' ),
-	PostTime = require( 'reader/post-time' ),
-	utils = require( 'reader/utils' ),
-	stats = require( 'reader/stats' );
-
-var PostByline = React.createClass( {
-
-	recordTagClick: function() {
-		stats.recordAction( 'click_tag' );
-		stats.recordGaEvent( 'Clicked Tag Link' );
-		stats.recordTrackForPost( 'calypso_reader_tag_clicked', this.props.post, {
+import ExternalLink from 'components/external-link';
+import Gravatar from 'components/gravatar';
+import Gridicon from 'components/gridicon';
+import PostTime from 'reader/post-time';
+import { siteNameFromSiteAndPost } from 'reader/utils';
+import {
+	recordAction,
+	recordGaEvent,
+	recordTrackForPost,
+	recordPermalinkClick
+ } from 'reader/stats';
+
+class PostByline extends React.Component {
+
+	constructor( ) {
+		super( );
+		this.recordTagClick = this.recordTagClick.bind( this );
+		this.recordAuthorClick = this.recordAuthorClick.bind( this );
```

Comment:
I'm curious about why your binding these two functions to `this` when they should are already be bound correctly. Maybe I'm missing something. If its to satisfy some condition where `this` is lost when passing the function as a callback somewhere else then just passing `() => this.recordTagClick()` should suffice.